deps.bzl

```diff
@@ -6400,9 +6400,10 @@ def install_go_mod_dependencies(workspace_name = "buildbuddy"):
     )
     go_repository(
         name = "org_golang_google_protobuf",
+        build_file_proto_mode = "disable_global",
```

Comment:
mind throwing a comment in here explaining what we discussed offthread?

Reply:
Added a comment clarifying why this was needed and when we could remove it.